froi/io/xml_api.py

```diff
@@ -0,0 +1,71 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+
+import os
+import xml.dom.minidom
+import nibabel as nib
+
+def xmlread_label(xmlname, tagname, index):
+    """
+    read specific label from xml file according to tag and index.
+
+    """
+    parent_path = os.path.dirname(os.getcwd())
+    tar_path = parent_path+'/froi/data/atlas/'
```

Comment:
The `tar_path` does not need to generated in the function, it should be got as an argument.